packages/utils/src/use-focus-change.ts

```diff
@@ -22,20 +22,24 @@ export function useFocusChange(
 		lastActiveElement.current = ownerDocument.activeElement;
 
 		function onChange(event: FocusEvent) {
-			if (lastActiveElement.current !== ownerDocument.activeElement) {
-				handleChange(
-					ownerDocument.activeElement,
-					lastActiveElement.current,
-					event
-				);
-				lastActiveElement.current = ownerDocument.activeElement;
+			if (
+				when === "focus" &&
```

Comment:
If do not check it and try use this hook with `blur`, handleChange will never be called, because `ownerDocument.activeElement` will be always `ownerDocument.body`.